CvGameCoreDLL_Expansion2/CvUnit.cpp

```diff
@@ -20404,10 +20406,10 @@ void CvUnit::setXY(int iX, int iY, bool bGroup, bool bUpdate, bool bShow, bool b
 										bool bDoCapture = false;
 #if defined(MOD_BALANCE_CORE)
 										bool bDoEvade = false;
-										if (pLoopUnit->IsCivilianUnit() && pLoopUnit->getExtraWithdrawal() > 0 && pLoopUnit->CanFallBack(*this, true))
+										if (pLoopUnit->IsCivilianUnit() && pLoopUnit->CheckWithdrawal(*this))
```

Comment:
The situation relating to this line of code deals with a military unit trying to capture/kill an enemy civilian unit by entering its square.

If the will of VP is to always have units with withdraw chance work 100% the first time, then it should work the same way here.

Reply:
It does work the same way. In MOD_BALANCE_VP, `CheckWithdrawal` always returns true on the first attack and false on subsequent ones.

Reply:
Right; missed the `GetWithdrawChance()` inside `CheckWithdrawal()`